### PR TITLE
add Australia vaccination data from covidlive.com.au

### PR DESCRIPTION
**Pull request by dhan16**
details at Issue  #443

### CODE REVIEW

File: src/lib/vaccinations_utils.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pandas import DataFrame
+
+
+def add_total_persons_vaccinated_estimate(data: DataFrame):
```
Comment (owahltinez) [inline src/lib/vaccinations_utils.py:18]: Can you make a copy instead of editing the data inplace?

**Review by owahltinez — CHANGES_REQUESTED** [review]
One small change requested, other than that LGTM

File: src/lib/vaccinations_utils.py (hunk 1 above)
Comment (dhan16) [reply src/lib/vaccinations_utils.py]: thanks for reviewing!

I've done it slightly differently..the function now returns a series and no longer edits the data inplace

so now its upto the caller. in the two callers I still edit inlace like below
data["total_persons_vaccinated"] = estimate_total_persons_vaccinated(data)

that should be OK as there are other inlace edits in the datasources 

WDYT?

**Review by owahltinez — APPROVED** [review]
LGTM, thanks!

**Pull request merged**